README.md

```diff
@@ -4,30 +4,30 @@
 [![GitHub Tag](https://img.shields.io/github/v/tag/TrueBrain/actions-flake8?include_prereleases&label=stable)](https://github.com/TrueBrain/actions-flake8/releases)
 [![GitHub commits since latest release](https://img.shields.io/github/commits-since/TrueBrain/actions-flake8/latest/main)](https://github.com/TrueBrain/actions-flake8/commits/main)
 
-This GitHub Actions runs flake8 over your code.
+This GitHub Actions runs [`flake8`](https://github.com/PyCQA/flake8) over your code.
```

Comment:
Please don't add ` around flake8. It is already getting a highlight as it is a link.